daemon/cli.c

```diff
@@ -223,6 +227,18 @@ static void cli_incoming_list_totals(str *instr, struct streambuf *replybuffer)
 		(unsigned long long)delete_iv.time_max.tv_sec,(unsigned long long)delete_iv.time_max.tv_usec,
 		(unsigned long long)delete_iv.time_avg.tv_sec,(unsigned long long)delete_iv.time_avg.tv_usec);
 
+	streambuf_printf(replybuffer, " Min/Max/Avg offer requests per second           :%llu/%llu/%llu per sec\n",
+		(unsigned long long)offers_ps.ps_min,
+		(unsigned long long)offers_ps.ps_max,
+		(unsigned long long)offers_ps.ps_avg);
+	streambuf_printf(replybuffer, " Min/Max/Avg answer requests per second          :%llu/%llu/%llu per sec\n",	(unsigned long long)answers_ps.ps_min,
+		(unsigned long long)answers_ps.ps_max,
+		(unsigned long long)answers_ps.ps_avg);
+	streambuf_printf(replybuffer, " Min/Max/Avg delete requests per second          :%llu/%llu/%llu per sec\n",
+		(unsigned long long)deletes_ps.ps_min,
+		(unsigned long long)deletes_ps.ps_max,
+		(unsigned long long)deletes_ps.ps_avg);
```

Comment:
Aren't these missing the division `avg / count` ? Or is this intentional?

Reply:
This is intentional, the average is computed and saved in the `send_graphite_data` function. Dividing again by the count will mean that it is done twice and it is incorrect. This is also happening for the statistics for the average time for offer/answer/delete. I will address this issue in a new commit